src/main/java/com/mraof/minestuck/computer/editmode/DeployList.java

```diff
@@ -38,27 +39,53 @@
+		registerItem("card_punched_card", new GristSet(GristTypes.BUILD, 25), null, 0, config(MinestuckConfig.SERVER.deployCard), (sburbConnection, world) -> AlchemyHelper.createCard(new ItemStack(MSItems.CAPTCHA_CARD.get()), true), false);
+		
+		//Atheneum
+		registerItem("stone", new ItemStack(Blocks.STONE), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("cobblestone", new ItemStack(Blocks.COBBLESTONE), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("oak_planks", new ItemStack(Blocks.OAK_PLANKS), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("oak_slab", new ItemStack(Blocks.OAK_SLAB), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("oak_stairs", new ItemStack(Blocks.OAK_STAIRS), new GristSet(GristTypes.BUILD, 2), 0, true);
+		registerItem("oak_fence", new ItemStack(Blocks.OAK_FENCE), new GristSet(GristTypes.BUILD, 2), 0, true);
+		registerItem("oak_log", new ItemStack(Blocks.OAK_LOG), new GristSet(GristTypes.BUILD, 4), 0, true);
+		registerItem("ladder", new ItemStack(Blocks.LADDER), new GristSet(GristTypes.BUILD, 16), 0, true);
+		registerItem("birch_planks", new ItemStack(Blocks.BIRCH_PLANKS), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("spruce_planks", new ItemStack(Blocks.SPRUCE_PLANKS), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("dark_oak_planks", new ItemStack(Blocks.DARK_OAK_PLANKS), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("acacia_planks", new ItemStack(Blocks.ACACIA_PLANKS), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("jungle_planks", new ItemStack(Blocks.JUNGLE_PLANKS), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("birch_log", new ItemStack(Blocks.BIRCH_LOG), new GristSet(GristTypes.BUILD, 4), 0, true);
+		registerItem("spruce_log", new ItemStack(Blocks.SPRUCE_LOG), new GristSet(GristTypes.BUILD, 4), 0, true);
+		registerItem("dark_oak_log", new ItemStack(Blocks.DARK_OAK_LOG), new GristSet(GristTypes.BUILD, 4), 0, true);
+		registerItem("acacia_log", new ItemStack(Blocks.ACACIA_LOG), new GristSet(GristTypes.BUILD, 4), 0, true);
+		registerItem("jungle_log", new ItemStack(Blocks.JUNGLE_LOG), new GristSet(GristTypes.BUILD, 4), 0, true);
+		registerItem("andesite", new ItemStack(Blocks.ANDESITE), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("diorite", new ItemStack(Blocks.DIORITE), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("granite", new ItemStack(Blocks.GRANITE), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("polished_andesite", new ItemStack(Blocks.POLISHED_ANDESITE), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("polished_diorite", new ItemStack(Blocks.POLISHED_DIORITE), new GristSet(GristTypes.BUILD, 1), 0, true);
+		registerItem("polished_granite", new ItemStack(Blocks.POLISHED_GRANITE), new GristSet(GristTypes.BUILD, 1), 0, true);
```

Comment:
Doesn't seem right to lump these together with the deployable stuff. How about making a separate list instead?
Should the atheneum items really have a hardcoded cost, rather than relying on the recipe-based grist cost?
Also, deployable items will lock in the connection. The same shouldn't apply to atheneum items, right?

Reply:
> I thought that the deployable items and atheneum items had similar enough functions that they could be lumped together in the same list.

I'm not so sure about that myself.

> The atheneum items purposefully have a hardcoded cost so that they have a different cost when building and destroying them in editmode, as opposed to when alchemizing or widgeting them normally. It's so that things like ladders can be made more expensive, while things like basic building blocks can be made cheaper.

That seems a bit strange to me, and I'm not really seeing the purpose behind it. Has a discussion happened on it previously in the discord that I can check out?

> Why do deployable items lock in the connection?? I was not aware of that. Wouldn't that cause the connection to be locked in the moment you enter editmode?

The point has been to lock a connection at the latest reasonable time. This is not when the conntection is made, and it's not when the server player enters editmode for the first time, but when the server player starts placing/dropping things that lets the player enter. That happens when a deployable item/block is dropped/placed. (And happens at the call of `SburbHandler.giveItems()`)

Reply:
> That seems a bit strange to me, and I'm not really seeing the purpose behind it. Has a discussion happened on it previously in the discord that I can check out?

Yes, in the Sburb Editmode Enhancements thread in the coding channel. I'll pin it since it's a bit buried.